cmd/agent/dist/conf.d/network.d/conf.yaml.default

```diff
@@ -1,33 +1,91 @@
 
 init_config:
 
+## The Network check only supports one configured instance.
+
 instances:
-  # Network check only supports one configured instance
+
+    ## @param collect_connection_state - boolean - required
+    ## Set to true to collect connection states for your interfaces.
+    #
   - collect_connection_state: false
-    excluded_interfaces:
-      - lo
-      - lo0
-    # Optionally completely ignore any network interface
-    # matching the given regex:
-    # excluded_interface_re: my-network-interface.*
-
-    # Do not combine connection states
-    # By default we combine states like fin_wait_1 and fin_wait_2
-    # together into one state: 'closing'
-    # For some people, this is fine, but others need more granular data
-    # enable this option to get more granular data
-    # combine_connection_states: no
-
-    # By default, most metrics are submitted as rates.
-    # However, some metrics like tcp/udp retransmissions and errors are
-    # better handled as counts.
-    # You can choose to enable either or both types of metrics.
-    # Count metrics will have '.count' added to the metric name.
+
+    ## @param excluded_interfaces - list of strings - optional
+    ## List of interfaces to exclude from the check.
+    #
+    # excluded_interfaces:
+    #   - lo
+    #   - lo0
+
+    ## @param excluded_interface_re - string - optional
+    ## Completely ignore any network interface matching the given regex.
+    #
+    # excluded_interface_re: <NETWORK_INTERFACE_NAME>.*
+
+    ## @param combine_connection_states - boolean - optional - default: true
+    ## Set to false to prevent combination of connection states.
+    ## By default, states like fin_wait_1 and fin_wait_2 are combined
+    ## together into one state: 'closing' disables this option to get more granular data.
+    #
+    # combine_connection_states: true
+
+    ## @param collect_rate_metrics - boolean - optional - default: true
+    ## By default, most metrics are submitted as rates.
+    ## However, some metrics like TCP/UDP retransmissions and errors are
+    ## better handled as counts.
+    ## Choose to disable rate metrics by setting collect_rate_metrics to false.
+    #
     # collect_rate_metrics: true
+
+    ## @param collect_count_metrics - boolean - optional - default: false
+    ## By default, most metrics are submitted as rates.
+    ## However, some metrics like TCP/UDP retransmissions and errors are
+    ## better handled as counts.
+    ## Choose to enable count metrics by setting collect_count_metrics to true.
+    ## Count metrics have '.count' added to the metric name.
+    #
     # collect_count_metrics: false
 
+    ## @param conntrack_path - string - optional
+    ## Linux only.
+    ## The location of the conntrack executable for collecting stats from conntrack -S.
+    ## This is run with sudo, so an entry needs to be added to the sudoers file.
+    ## By default, these metrics are not sent.
+    #
+    # conntrack_path: /usr/sbin/conntrack
```

Comment:
same here than for the disk check: some option are only available in the Python version of the check.

Reply:
Same here